client/dashboard/components/logs-activity/activity-event.scss

```diff
@@ -19,3 +19,7 @@
 		color: $gray-700;
```

Comment:
@keoshi with your changes this color is getting picked up also for the title

<img width="1160" height="432" alt="Image" src="https://github.com/user-attachments/assets/f318f86b-a33c-4790-8bf8-b88f9fde3d01" />

I think, at this point, it could make sense to have a dedicated class for the `formattedContent` too, what do you think?

Reply:
Sounds like a great idea to future-proof it. Would you like me to make that change in this PR?